setup.py

```diff
@@ -72,8 +72,7 @@ class CustomDist(sdist):
     def run(self):
         copy_tree(f'src/main/proto/{package_name}', package_name)
 
-        copy_tree(f'src/gen/main/python/{package_name}', package_name)
-        copy_tree(f'src/gen/main/services/python/{package_name}', package_name)
+        copy_tree(f'build/generated/source/proto/main/services/python/{package_name}', package_name)
```

Comment:
Please, check that all required files are in the final archive

Reply:
yes, all files are there